_config.yml

```diff
@@ -21,12 +22,30 @@ collections:
     output: true
     permalink: /support/
 
+markdown: kramdown
+kramdown:
+  input: GFM
+  syntax_highlighter: rouge
+
+  syntax_highlighter_opts:
+    css_class: 'rougeHighlight'
+    span:
+      line_numbers: false
+    block:
+      line_numbers: true
+      start_line: 1
+
 sass:
   style: :compressed
 
-# Currently unused, since we are on kramdown
 redcarpet:
-  extensions: [with_toc_data, smart]
+  extensions: [with_toc_data]
+
+# Gems
```

Comment:
Delete this comment? Or add a comment explaining why these are added. But their rough purpose seems clear from their names.